datasets/superb/superb.py

```diff
@@ -383,3 +439,26 @@ def _get_speakers(rec, data, args):
         }
         for segment in data.segments[rec]
     ]
+
+
+def _split_ks_files(archive_path, split):
+    audio_path = os.path.join(archive_path, "**/*.wav")
+    audio_paths = sorted(glob.glob(audio_path))
```

Comment:
NIT: this `sorted` is redundant as we already call `sorted` in the return from this function: line 291 `for audio_file in sorted(splits[split]):`